src/types/library/library_item.rs

```diff
@@ -63,6 +63,11 @@ impl LibraryItem {
         }
     }
 
+    #[inline]
+    pub fn watched(&self) -> bool {
```

Comment:
please document the fn in order for the behavior to be visible in https://stremio.github.io/stremio-core/stremio_core/